_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -150,6 +150,17 @@ public static function register_endpoints() {
 			'permission_callback' => __CLASS__ . '::get_user_connection_data_permission_callback',
 		) );
 
+		// Get list of plugins that use the Jetpack connection.
+		register_rest_route(
+			'jetpack/v4',
+			'/connection/plugins',
+			array(
+				'methods'             => WP_REST_Server::READABLE,
+				'callback'            => __CLASS__ . '::get_connection_plugins',
+				'permission_callback' => __CLASS__ . '::view_admin_page_permission_check',
```

Comment:
Should we limit this to folks who can act on those plugins, maybe with something like `activate_plugins_permission_check`?

Reply:
Good idea, I changed the permission.